checkov/terraform/parser.py

```diff
@@ -292,7 +292,7 @@ def _internal_dir_load(self, directory: str,
         #                      because it break encapsulations and I don't want to cause confusion about what data
         #                      set it being processed. To avoid this, here's a Callable that will get the data
         #                      map for a particular module reference. (Might be OCD, but...)
-        module_data_retrieval = lambda module_ref: self.out_definitions.get(module_ref)
+        module_data_retrieval = lambda module_ref: self.out_definitions.get(module_ref)  # noqa
```

Comment:
@gruebel what's wrong with this line that it required `# noqa`?

Reply:
the `module_data_retrieval` variable is unused, was not sure, if I can just delete the whole line 😄

Reply:
If checks pass, it's good to go 🙏

Reply:
removed it 😄